code/game/objects/items/storage/belt.dm

```diff
@@ -532,7 +532,7 @@
 
 /obj/item/storage/belt/military/snack/Initialize(mapload)
 	. = ..()
-	var/sponsor = pick("Donk Co.", "Waffle Co.", "Roffle Co.", "Gorlax Marauders", "Tiger Cooperative")
+	var/sponsor = pick("Donk Co.", "Waffle Corp", "Roffle Co.", "Gorlax Marauders", "Tiger Cooperative")
```

Comment:
while you're at it could you change 'gorlax' to 'gorlex' since that's also an error